site/docs/params/params.md

```diff
@@ -2,7 +2,7 @@
 layout: docs
 menu: docs
 title: Parameter / Interaction
-permalink: /docs/parameter.html
+permalink: /docs/params.html
```

Comment:
@arvind we had to pages at the same url. Where should this page go? It's not linked from anywhere.

Reply:
It looks like this page was created in #7138, and merged before we had resolved the discussion about whether (and how) or not to unify params and selections. And, because it was merged into master, I didn’t have access to it in the selection-params feature branch (ergo the duplicate docs). The content below describing example param definitions should be merged into parameter.md. Will try to send a PR later today if no one gets to it by then.

Reply:
I just merged them.

Reply:
🙌